config/apcupsd/apcupsd_status.php

```diff
@@ -28,6 +28,7 @@
 */
 
 require("guiconfig.inc");
+require_once("/usr/local/pkg/apcupsd.inc");
```

Comment:
/usr/local/pkg is on php include path, you can drop it here and just use filename